style.css

```diff
@@ -0,0 +1,106 @@
+.wrapper {
```

Comment:
- [ ] You have done so well implementing the design as per the figma template, but looking at the `right margin`, there's quite a remarkable difference from the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A988). Please also try to match the colors and font sizes. 
- Your design 
![image](https://user-images.githubusercontent.com/62849872/143445170-a1ec7cce-8e85-4c88-a441-1924850961e7.png)

- Figma design 
![image](https://user-images.githubusercontent.com/62849872/143445284-244646ef-fec8-440b-84dd-f4e6689bea3e.png)

I think you need to review the properties applied to the `wrapper` class. Comment out the wrapper class and see the display of your page in the browser and you will have an idea how to fix it.

Reply:
Hello @t-yanick. Am happy with your reviews and I promise you that I am going to make changes that you have told me to do as soon as possible. Thank you.